src/main/java/com/google/api/codegen/transformer/py/PythonSurfaceNamer.java

```diff
@@ -146,8 +146,8 @@ public String getParseFunctionName(String var, SingleResourceNameConfig resource
 
   @Override
   public String getGrpcClientTypeName(Interface apiInterface) {
-    String fullName = getModelTypeFormatter().getFullNameFor(apiInterface) + "Stub";
-    return getTypeNameConverter().getTypeName(fullName).getNickname();
+    String fullName = getModelTypeFormatter().getFullNameFor(apiInterface) + "Client";
```

Comment:
This change is actually to make it such that the terminology is intuitive. The grpcClientTypeName is the type of the grpcClient. The stubType is the type of the grpcClientStub.